src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -883,16 +883,10 @@ function startsWithSvChartType(chartType?: string) {
 }
 
 export function getMolecularProfileIdsFromUniqueKey(uniqueKey: string) {
-    const parts = uniqueKey.split(UNIQUE_KEY_SEPARATOR);
-    if (!startsWithSvChartType(parts[0])) {
-        return parts;
-    } else {
-        return _.map(
-            parts,
-            molecularProfileId =>
-                molecularProfileId.split(CHART_TYPE_SEPARATOR)[1]
-        );
-    }
+    return (startsWithSvChartType(uniqueKey)
```

Comment:
can you put a comment with the format of the key and some explaination of why it sometimes but not always has chartype?